src/styles/main.scss

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.container {
+  padding-top: 100px;
+  display: flex;
+  justify-content: center;
+}
+
+$columns: 7;
+$cell-size: 100px;
+$padding: 10px;
+$gap: 1px;
+$week-days:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sat 5,
+  sun 6;
+
+.calendar {
+  width: ($cell-size * $columns) + ($gap * ($columns - 1)) + ($padding * 2);
```

Comment:
You can transfer mathematical operations to separate variables and use the result immediately, this will increase readability.

Reply:
That's a good point, thanks.